x-pack/plugins/enterprise_search/public/applications/app_search/components/engine/engine_nav.tsx

```diff
@@ -103,7 +103,11 @@ export const EngineNav: React.FC = () => {
         {OVERVIEW_TITLE}
       </SideNavLink>
       {canViewEngineAnalytics && (
-        <SideNavLink to={engineRoute + ENGINE_ANALYTICS_PATH} data-test-subj="EngineAnalyticsLink">
+        <SideNavLink
+          to={engineRoute + ENGINE_ANALYTICS_PATH}
+          shouldShowActiveForSubroutes={true}
```

Comment:
This was a super awesome and easy prop to add to get active highlighting working - thanks for this initial work @JasonStoltz!